subsystem/authz_test.go

```diff
@@ -100,7 +100,12 @@ var _ = Describe("test authorization", func() {
 		})
 	})
 
-	Context("regular user", func() {
+	Context("regular user - tenancy disabled", func() {
+		if Options.EnableOrgTenancy {
```

Comment:
These tests should succeed even under tenancy access rules. We should see what is wrong here

Reply:
Yes but there are a few tests that should fail in case of tenancy enabled. E.g. "can't get not owned cluster" - when tenancy is enabled a user can get not owned cluster if in same org. So need to go over these and change/add ones specifically for tenancy support.